string-manipulation-1/src/get-words.js

```diff
@@ -0,0 +1,32 @@
+/* exported getWords */
+
+// create storage for the output
+// create storage for characters in current word
+// if the string does not contain anything,
+// then return an empty output.
+// look at each character in the string.
+// if there is a space,
+// then add the current word into the output.
+// clear the storage for the current word.
+// otherwise,
+// add the current character into the storage for current word.
```

Comment:
It's usually a good idea to indent your pseudocode the same way you would your code, just so that it's clear where loop and conditional blocks begin and end, etc.